rpcd/playbooks/roles/rpc_maas/templates/disk_utilisation.yaml.j2

```diff
@@ -1,8 +1,10 @@
+{% set custom_maas_check_period = maas_check_period_override | selectattr('label', 'equalto', 'disk_utilisation') | map(attribute='value') %}
```

Comment:
Is there not a cleaner dict access syntax in Jinja? Would this have to be repeated as boilerplate in every check template?